wiki/fabrication/ultimaker-series.md

```diff
@@ -0,0 +1,100 @@
+
+
+## Typical Failures
+If you are encountering some issues with executing your print, here are some common failure points that we have come across in our use of the Ultimaker 3/S3.
+
+### Not enough material
+Starting with an easy one, your print might stop if your spool runs out of material. Fortunately, the interface does well to walk you through the process of changing the material on the back. For more information, you can learn more by watching this video on [changing the filament spool](https://youtu.be/c2XznbVFOBs) (Note: this video shows the Ultimaker 3, which does not have the touch screen, but the process and prompts are identical on the S3).
+
+## Wrong print settings
+Your print doesn't start because you configured your Cura screen wrong! Go back to your PREPARE page and check the settings at Circle 3 in Figure 1.
+
+### The wrong core to the wrong material
+Often forgotten, it is very important to have the correct print core for each material. The print core is the piece of hardware that heats the filament before it used to 'print' your part. Different filaments have different requirements for nozzle size and the temperature they must be heated to before printing. If you use a mismatched print core, you will quickly block and damage the print core such that it is rendered completely unusable.
+
+**Print cores are not cheap**, so try to avoid this by always carefully checking the specs for your filament to make sure you match the appropriate filament with the appropriate print core. [Learn more about print cores](https://ultimaker.com/3d-printers/ultimaker-print-cores-and-addons).
+
+### 'Nozzle Offset Probe Failed'
+This one is a difficult one. Even with extensive research, it has been hard to find a solution that consistently works. We recommend you check the online forums, such as [this one](https://community.ultimaker.com/topic/35521-nozzle-offset-probe-failed/). Other steps you can try include:
+
+- Reload the filament at the back of the printer.
+- Check that the build plate is correctly centered and secured down.
+- Clean the build plate to ensure there is no undue residue.
+- Clean and/or reload the print cores
+
+
+### Sometimes, things just go wrong.
+Sometimes, things just go wrong. Figure 3 has an example of a very long print that went wrong. This one was due to the support filament not extruding correctly, leading to a failure to print the overhang. This one was actually due to a failure to use the correct print core for the support material (PVA, in this case). So when the print core got jammed with incorrectly melted PVA, the print failed. This just goes to show how important it is to pay attention to some of the simple rules outlined on this page, and you can avoid most failure likes this one.
+
+<figcaption align = "center"><b>Figure 3: Prints can fail. It happens! </b></figcaption>
+
+![Ultimaker Cura Screen](assets/ultimaker_fig3.jpg)
```

Comment:
@HardikSingh97 This image link is broken - please update (extensions are case-sensitive!).